apps/builder/app/builder/features/style-panel/style-source/style-source-control.tsx

```diff
@@ -318,14 +324,15 @@ export const StyleSourceControl = ({
         disabled={disabled}
         aria-current={selected && state === undefined}
         role="button"
```

Comment:
i don't think role=button should be here

Reply:
me neither, this entire component is badly made, no idea what we were thinking